README.md

```diff
@@ -18,6 +19,15 @@ displaying information from those in a centralized place.
 
 ---
 
+## Cloning and getting into the Project Dir
+
+  To clone this repository simply type
+
+    $ cd ~ && mkdir workspace && cd workspace && git clone https://github.com/mozilla/delivery-dashboard && cd delivery-dashboard
```

Comment:
This is a bit too much helping :)

    git clone https://github.com/mozilla/delivery-dashboard

Should be sufficient IMHO